src/hal/stm32f1/iomem.ld

```diff
@@ -0,0 +1,37 @@
+stm32f1_iomem_PWR   = 0x40007000;
```

Comment:
It looks like stm32f1 has proper svd files: https://github.com/posborne/cmsis-svd/tree/master/data/STMicro, that should be the preferred method to generate ioregs now, as `extern` registers and `iomem.ld` is now deprecated, replaced by placement ioregs.